executor/src/main/java/com/redhat/service/bridge/executor/ExecutorsService.java

```diff
@@ -31,49 +29,37 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ExecutorsService.class);
 
-    private Map<String, Set<Executor>> bridgeToProcessorMap = new HashMap<>();
-
     @Inject
-    ExecutorFactory executorFactory;
+    ExecutorConfigProvider executorConfigProvider;
 
     public void init(@Observes StartupEvent ev) {
         ExtensionProvider.getInstance().registerExtension(BridgeCloudEventExtension.class, BridgeCloudEventExtension::new);
     }
 
-    public void createExecutor(ProcessorDTO processorDTO) {
-
-        Executor executor = executorFactory.createExecutor(processorDTO);
-
-        synchronized (bridgeToProcessorMap) {
-            Set<Executor> executors = bridgeToProcessorMap.get(processorDTO.getBridge().getId());
-            if (executors == null) {
-                executors = new HashSet<>();
-            }
-
-            executors.add(executor);
-            bridgeToProcessorMap.put(processorDTO.getBridge().getId(), executors);
-        }
-    }
-
     @Incoming(EVENTS_IN_TOPIC)
-    public void processBridgeEvent(String event) {
-
+    public CompletionStage<Void> processBridgeEvent(final Message<String> message) {
         try {
-            CloudEvent cloudEvent = CloudEventUtils.decode(event);
+            CloudEvent cloudEvent = CloudEventUtils.decode(message.getPayload());
             BridgeCloudEventExtension bridgeCloudEventExtension = ExtensionProvider.getInstance().parseExtension(BridgeCloudEventExtension.class, cloudEvent);
-            Set<Executor> executors = bridgeToProcessorMap.get(bridgeCloudEventExtension.getBridgeId());
+            String bridgeId = bridgeCloudEventExtension.getBridgeId();
+            Set<Executor> executors = executorConfigProvider.getExecutors(bridgeId);
+            if (executors == null) {
+                LOG.info("[executor] A message for BridgeID {} has been received, but no executors were found.", bridgeId);
+            }
             if (executors != null) {
                 for (Executor e : executors) {
                     try {
                         e.onEvent(cloudEvent);
                     } catch (Throwable t) {
                         // Inner Throwable catch is to provide more specific context around which Executor failed to handle the Event, rather than a generic failure
-                        LOG.error("Processor with id '{}' on bridge '{}' failed to handle Event.", e.getProcessor().getId(), e.getProcessor().getBridge().getId(), t);
+                        LOG.error("Processor with id '{}' on bridge '{}' failed to handle Event. The message is acked anyway.", e.getProcessor().getId(), e.getProcessor().getBridge().getId(), t);
```

Comment:
Let's keep `[executor]` prefix also here. Wdyt?
```suggestion
                        LOG.error("[executor] Processor with id '{}' on bridge '{}' failed to handle Event. The message is acked anyway.", e.getProcessor().getId(), e.getProcessor().getBridge().getId(), t);
```